### DISCUSSION TITLE
How can i use lib in my react-native tv app

### DISCUSSION THREAD

**Discussion by vishalhyperspace** [LunatiqueCoder/react-native-media-console#54] [Q&A]
Hello 
I have tried to install react-natve-media-console@next and tested in my tv app. but controls not working in my tv app. 
here is installed version in my app
`    "react-native-media-console": "^2.0.2-0",
`
please help

**Comment by LunatiqueCoder** [comment]
Hello! Why `@next`? Specifying `@next` will install the latest release candidate. Currently there is no release candidate, the latest version is 2.0.6. 

`npm install react-native-media-console`

or 

`yarn add react-native-media-console`
